images/base/files/usr/local/bin/entrypoint

```diff
@@ -266,6 +266,7 @@ fix_product_name() {
     echo 'INFO: faking /sys/class/dmi/id/product_name to be "kind"' >&2
     echo 'kind' > /kind/product_name
     mount -o ro,bind /kind/product_name /sys/class/dmi/id/product_name
+    jq '.mounts[.mounts | length] |= . + {"destination": "/sys/class/dmi/id/product_name", "source": "/kind/product_name", "options": ["ro", "bind"]}' /etc/containerd/cri-base.json > /tmp/cri-base.json && mv /tmp/cri-base.json /etc/containerd/cri-base.json
```

Comment:
https://github.com/kubernetes-sigs/kind/pull/2321/files#r668631946
following up on this comment here, is this value not always the same?
```
    {
      "destination": "/sys/class/dmi/id/product_name",
      "source": "/kind/product_name",
      "options": [
        "ro",
        "bind"
      ]
    }
```

Reply:
ahh, you right, are we just that generting it is not needed ?

Reply:
that is my question :sweat_smile: